DashboardEmbedding/EmbedDashboardswithSSOusingToken/Program.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EmbedDashboardswithSSOusingToken
+{
+    public class Program
+    {
+        static void Main(string[] args)
+        {
+            // Variable declaration to form the embed URL 
+            var dashboardUrl = "https://onpremise-demo.boldbi.com/bi/site/site2/dashboards/9b5e9121-1379-4724-afba-c9674eb4f867/Education/adventure%20datasource%20dashboard?"; // URL of the dashboard to be embedded. Add '? or &' at the end of the URL based on your dashboard URL.
```

Comment:
You shouldn't use the Onpremise-demo server link here. Give dummy URL here.

Reply:
Dummy url added.